scale/docs/rest/v6/job_type.rst

```diff
@@ -418,193 +422,195 @@ Response: 200 OK
 | created                  | ISO-8601 Datetime | When the associated database model was initially created.                |
 +--------------------------+-------------------+--------------------------------------------------------------------------+
 
-.. _rest_job_type_create:
+.. _rest_v6_add_seed_image:
+
+v6 Add Seed Image
+-----------------
+
+**Example POST /v6/job-types/ API call**
+
+Request: POST http://.../v6/job-types/
+
+ .. code-block:: javascript
+     {
+      "icon_code": "f1c5",
+      "max_scheduled": 1,
+      "docker_image": "test-1.0.0-seed:1.0.0",
+      "configuration": {
+        "version": "2.0",
```

Comment:
The API should accept version 6 of the configuration. The only change here is to omit the `version` field.

Reply:
Should it be removed or just changed to 6 as these are POST examples?

Reply:
Just remove the `"version": "2.0",` line, since the rest is still version 6 compliant.